UnitsNet.Tests/CustomCode/LengthTests.cs

```diff
@@ -55,6 +55,12 @@ public class LengthTests : LengthTestsBase
 
         protected override double YardsInOneMeter => 1.09361;
 
+        private const double fathomsInOneMeter = 1.0 / 1.8288;
+        protected override double FathomsInOneMeter => fathomsInOneMeter;
+
+        private const double shacklesInOneMeter = 1.0 / 27.432;
+        protected override double ShacklesInOneMeter => shacklesInOneMeter;
```

Comment:
Please remove the fields to follow the code style.
Also it is important that these values are not calculations, but constant values looked up in some source of truth (google, wikipedia, online converters, books etc.). This is to serve as a double check of the calculations in the `.json` file.
I usually just google the name of the property and the value often shows up directly in the google calculator:
https://www.google.no/search?q=FathomsInOneMeter&oq=FathomsInOneMeter&aqs=chrome..69i57.808j0j9&sourceid=chrome&ie=UTF-8